source/io/EventLoopGroup.cpp

```diff
@@ -2,8 +2,11 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
+#include <aws/crt/Types.h>
```

Comment:
Feels like all the new includes should be removable given the migration to ApiHandle